docker/package/package_generator.py

```diff
@@ -193,15 +194,18 @@ def gen_install(pkg: Package, out):
 
 packages = [
     Package("tezos-client",
-            "CLI client for interacting with tezos blockchain"),
+            "CLI client for interacting with tezos blockchain",
+            optional_opam_deps = ["tls", "ledgerwallet-tezos"]),
     Package("tezos-admin-client",
-            "Administration tool for the node") ,
+            "Administration tool for the node",
+            optional_opam_deps = ["tls"]),
     Package("tezos-node",
```

Comment:
Can you run a node accessible via https if you don't explicitly require `tls` for it?

Reply:
tezos-node have tls as an explicit dependency